pysparkling/sql/session.py

```diff
@@ -261,6 +260,9 @@ def prepare(obj):
             def prepare(obj):
                 return obj
 
+        # pylint: disable=import-outside-toplevel, cyclic-import
+        from ..rdd import RDD
```

Comment:
My understanding is that the import structure within the `sql` module is complicated and not great at the moment. However, the core classes outside of the `sql` module should be fine. Do you mind re-checking whether these imports outside of top-level are really necessary for those cases like this one?

Reply:
I was able to move a few, but the complete cleanup of these imports would lead me too far. I'm going to move it into an issue (as it very likely will become) to rationalize this later on.